### PR TITLE
Add firefox/browsers/chromebook (#9490 Fixes #9512, #9515)

### LINKED ISSUE #9512: Epic: Chromebook page
## Description

We want to build a page for chromebook

url: /firefox/browsers/chromebook/

Pitch: https://docs.google.com/document/d/17aL69fIIUhftGhFnx87OcfYcWoecs-xno1a1rT6_Nc4
Gdrive: https://drive.google.com/drive/folders/1VFoA7IJlj_jUYWkGIp4kYhMGHRjzJ-k_

---

#### Issues
- [x] Content https://github.com/mozilla/bedrock/issues/9513
- [x] Design  https://github.com/mozilla/bedrock/issues/9514
- [ ] Coding https://github.com/mozilla/bedrock/issues/9515

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description
Adds firefox/browsers/chromebook

## Issue / Bugzilla link
#9490
#9512
#9515

## Testing
[Copy](https://docs.google.com/document/d/19Vin0P4F8w2FdeXGl8Df6wSLIzD2bKPAcD5XRcLTc-0/)
[Design](https://github.com/mozilla/bedrock/issues/9514#issuecomment-706293652) -- _design reflects older menu list design_

### CODE REVIEW

**Comment by amychurchwell** [comment]
Marking as "Do Not Merge" until demo received approvals from Raphael & Chad

**Comment by amychurchwell** [comment]
Approved!

File: bedrock/firefox/templates/firefox/browsers/chromebook.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    <h4 class="mzp-c-menu-list-title">{{ ftl('dropdown-copy') }}</h4>
+    <ul class="mzp-c-menu-list-list download-platform-list">
+      <li class="mzp-c-menu-list-item">
+        <a href="{{ firefox_adjust_url('android', 'firefox-browsers-faq') }}" rel="external noopener" data-link-type="download" data-display-name="Android" data-download-version="android" data-download-os="Android">
+          {{ ftl('get-firefox-for') }}
+          <p><small>{{ ftl('x86-based-chromebook') }}</small></p>
+        </a>
+      </li>
+      <li class="mzp-c-menu-list-item t-sumo">
+        <a href="https://support.mozilla.org/en-US/kb/run-firefox-chromeos?utm_source=www.mozilla.org-firefox-browsers-chromebook&amp;utm_medium=referral&amp;utm_campaign=seo" data-display-name="Get Firefox Desktop for Chromebook">
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/browsers/chromebook.html:39]: Can you double-check the analytics on this link, please?

File: bedrock/firefox/templates/firefox/browsers/chromebook.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    <div class="mzp-l-content mzp-l-card-half">
+      <div class="mzp-c-card">
+        <img src="{{ static('img/firefox/browsers/chromebook/mobile.png') }}" class="mzp-c-billboard-image" width="436" height="463" alt="">
+        <p>{{ ftl('install-firefox-from') }}</p>
+        {{ google_play_button(href=android_url, id='play-store-link') }}
+      </div>
+
+      <div class="mzp-c-card">
+        <img src="{{ static('img/firefox/browsers/chromebook/addon.png') }}" class="mzp-c-billboard-image" width="436" height="463" alt="">
+        <p>{{ ftl('install-firefox-as', url='https://support.mozilla.org/en-US/kb/run-firefox-chromeos?utm_source=www.mozilla.org-firefox-faq&amp;utm_medium=referral&amp;utm_campaign=seo') }}</p>
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/browsers/chromebook.html:79]: Do we want analytics on this link? I'd do it like this: https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/templates/firefox/browsers/compare/brave.html#L86

**Review by stephaniehobson — CHANGES_REQUESTED** [review]
Looking good; just needs a couple finishing touches.

The card images are larger than they need to be. Can you please create high and low res versions? The high res shouldn't be wider than 1200px. ([image sizes](https://protocol.mozilla.org/patterns/molecules/card.html) are defined in Protocol)

File: l10n/en/firefox/browsers/chromebook.ftl [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the { -brand-name-mozilla } Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/browsers/chromebook/
+
+# HTML page title
+
+get-firefox-browser = Get { -brand-name-firefox-browser } for { -brand-name-chromebook }
```
Comment (alexgibson) [inline l10n/en/firefox/browsers/chromebook.ftl:9]: Fluent IDs should use a page specific naming prefix: https://bedrock.readthedocs.io/en/latest/l10n.html#ftl-string-ids

File: l10n/en/brands.ftl [hunk 4]
```diff
@@ -129,6 +129,7 @@
 -brand-name-windows = Windows
 -brand-name-xp = XP
 -brand-name-vista = Vista
+-brand-name-chromeos = ChromeOS
```
Comment (alexgibson) [inline l10n/en/brands.ftl:132]: Should it be `Chrome OS`? https://www.google.com/intl/en_uk/chromebook/chrome-os/

File: l10n/en/firefox/browsers/chromebook.ftl [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+is-it-hard = Is it hard to install { -brand-name-firefox } for { -brand-name-chromebook }?
+we-wish-it = We wish it were more straightforward, but your { -brand-name-chromebook } would prefer to keep you in the { -brand-name-google-play } ecosystem. But, we think it’s worth the effort to install the { -brand-name-firefox } browser to your { -brand-name-chromebook } – and we’ve got resources to help you if you need them. There are two ways to get { -brand-name-firefox } on your device.
+install-firefox-from = Install { -brand-name-firefox } from { -brand-name-google-play } Store: on newer versions of { -brand-name-chromeos } (x86 based { -brand-name-chromebook } running { -brand-name-chromeos } 80 or later), you have the option to install the { -brand-name-firefox } for { -brand-name-android } app. This app is developed for mobile devices.
+
+# Variables:
+# $url (url) - link to https://support.mozilla.org/en-US/kb/run-firefox-chromeos
```
Comment (alexgibson) [inline l10n/en/firefox/browsers/chromebook.ftl:28]: Remove /en-US/

File: bedrock/firefox/templates/firefox/browsers/chromebook.html (hunk 2 above)
Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/browsers/chromebook.html:79]: I'm not sure, I just included it to be safe. Happy to remove if no one will use the info?

Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/browsers/chromebook.html:79]: asking Raphael.

File: l10n/en/firefox/browsers/chromebook.ftl (hunk 3 above)
Comment (amychurchwell) [reply l10n/en/firefox/browsers/chromebook.ftl:9]: adding `browsers-chromebook-` at the beginning of all IDs

File: bedrock/firefox/templates/firefox/browsers/chromebook.html [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+    <h4 class="mzp-c-menu-list-title">{{ ftl('browsers-chromebook-dropdown-copy') }}</h4>
+    <ul class="mzp-c-menu-list-list download-platform-list">
+      <li class="mzp-c-menu-list-item">
+        <a href="{{ firefox_adjust_url('android', 'firefox-browsers-faq') }}" rel="external noopener" data-link-type="download" data-display-name="Android" data-download-version="android" data-download-os="Android">
+          {{ ftl('browsers-chromebook-get-firefox-for') }}
+          <p><small>{{ ftl('browsers-chromebook-x86-based-chromebook') }}</small></p>
+        </a>
+      </li>
+      <li class="mzp-c-menu-list-item t-sumo">
+        <a href="https://support.mozilla.org/kb/run-firefox-chromeos" data-display-name="Get Firefox Desktop for Chromebook">
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/browsers/chromebook.html:38]: Not sure about the lone `data-display-name` property here without any others. I think we normally use this on download links?

File: bedrock/firefox/templates/firefox/browsers/chromebook.html [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+      <h2>{{ ftl('browsers-chromebook-is-it-hard') }}</h2>
+      <p>{{ ftl('browsers-chromebook-we-wish-it') }}</p>
+    </div>
+
+    <div class="mzp-l-content mzp-l-card-half">
+      <div class="mzp-c-card">
+        {{ high_res_img('img/firefox/browsers/chromebook/mobile.png', {'alt': '', 'class': 'mzp-c-billboard-image', 'width': '436', 'height': '464'}) }}
+
+        <p>{{ ftl('browsers-chromebook-install-firefox-from') }}</p>
+        {{ google_play_button(href=android_url, id='play-store-link') }}
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/browsers/chromebook.html:75]: You need to define android_url (I keep making this mistake myself 😆 )

`{% set android_url = firefox_adjust_url('android', 'firefox-daylight-promo-banner') %}`

**Review by stephaniehobson — CHANGES_REQUESTED** [review]
Updates are looking 💯 but I found something we should have caught in an earlier review.

**Comment by amychurchwell** [comment]
@stephaniehobson good catch! thank you. 👍

**Review by stephaniehobson — APPROVED** [review]
R+ 📘

**Pull request merged**